provisioning/service/tests/Config/TpmAttestationTests.cs

```diff
@@ -30,7 +31,7 @@ public void TpmAttestation_ThrowsOnNullEndorsementKey()
 
         [TestMethod]
         [TestCategory("DevService")]
-        public void TpmAttestation_ThrowsOnEnptyEndorsementKey()
+        public void TpmAttestation_ThrowsOnEmptyEndorsementKey()
```

Comment:
Missing test for null.

Reply:
`TpmAttestationTests` has a test for null EK `TpmAttestation_ThrowsOnNullEndorsementKey`